### PR TITLE
translate home page

### PR DESCRIPTION
**Pull request by awxiaoxian2020**

### CODE REVIEW

File: docs/typescriptlang/zh/index2.ts [hunk 1]
```diff
@@ -1,79 +1,79 @@
...
+  index_2_tab_2: "自动补全",
+  index_2_tab_3: "接口",
   index_2_tab_4: "JSX",
   index_2_tab_5: "ESNext",
 
   // Quick pitch
-  index_2_what_is: "What is TypeScript?",
-  index_2_what_is_js: "JavaScript and More",
+  index_2_what_is: "什么是 TypeScript？",
+  index_2_what_is_js: "比 JavaScript 更多",
```
Comment (hanyujie2002) [inline docs/typescriptlang/zh/index2.ts]: This looks somewhat weird 🤔 How about directly translate it as "JavaScript 以及更多"?

File: docs/typescriptlang/zh/index2.ts [hunk 2]
```diff
@@ -1,79 +1,79 @@
...
   index_2_migration_title: "TypeScript Testimonials",
-  index_2_migration_oss: "Open Source with TypeScript",
+  index_2_migration_oss: "TypeScript 开源项目",
 
   // Survey results
-  index_2_loved_by: "Loved by Developers",
-  index_2_loved_stack: `Voted <strong>2nd most loved programming language</strong> in the <so>Stack Overflow 2020 Developer survey</so>`,
-  index_2_loved_state_js: `TypeScript was <strong>used by 78%</strong> of the <js>2020 State of JS</js> respondents, with <strong>93% saying they would use it again</strong>.`,
-  index_2_loved_state_js2: `TypeScript was given the award for <strong>“Most Adopted Technology”</strong> based on year-on-year growth.`,
+  index_2_loved_by: "TypeScript 是开发者喜爱的语言",
```
Comment (hanyujie2002) [inline docs/typescriptlang/zh/index2.ts]:
```suggestion
  index_2_loved_by: "TypeScript 为开发者所喜爱"
```

File: docs/typescriptlang/zh/index2.ts [hunk 3]
```diff
@@ -1,79 +1,79 @@
...
+  index_2_loved_stack: `在 <so>Stack Overflow 2020 开发者调查</so>中，被选为<strong>第二受欢迎的编程语言</strong>。`,
```
Comment (hanyujie2002) [inline docs/typescriptlang/zh/index2.ts]:
```suggestion
  index_2_loved_stack: `在 <so>Stack Overflow 2020 开发者调查</so>中，是<strong>第二受欢迎的编程语言</strong>。`,
```

Also, ""

File: docs/typescriptlang/zh/index2.ts (hunk 1 above)
Comment (hanyujie2002) [inline docs/typescriptlang/zh/index2.ts]:
```suggestion
  index_2_what_is_js: "JavaScript 以及更多",
```

Comment (hanyujie2002) [inline docs/typescriptlang/zh/index2.ts]:
```suggestion
  index_2_what_is_js: "TypeScript 是 JavaScript 的超集",
```

Or this more direct one?

Comment (awxiaoxian2020) [reply docs/typescriptlang/zh/index2.ts]: The context is "TypeScript adds additional syntax to JavaScript to support a tighter integration with your editor. Catch errors early in your editor."

**Review by hanyujie2002 — APPROVED** [review]

**Pull request merged**